prov/rxm/src/rxm_init.c

```diff
@@ -38,8 +38,11 @@
 #include <ofi_prov.h>
 #include "rxm.h"
 
-#define RXM_ATOMIC_UNSUPPORTED_MSG_ORDER (OFI_ORDER_RAR_SET | OFI_ORDER_RAW_SET | \
-					  OFI_ORDER_WAR_SET | OFI_ORDER_WAW_SET | \
+// all bits excluding RMA specific bits
+#define RXM_ATOMIC_UNSUPPORTED_ORDER_SETS ((OFI_ORDER_RAR_SET | OFI_ORDER_RAW_SET | \
+                                          OFI_ORDER_WAR_SET | OFI_ORDER_WAW_SET) & \
+					  ~(OFI_ORDER_RMA_SET))
```

Comment:
We're getting deep into indirection here.  Can we define this more directly using the FI_ORDER_blah bits?

Reply:
Sure. My motivation for doing it the way that I did it was in the event that we added anything else -- we could just modify the top level defines rather than the sets. I am not opposed to just modifying the one define.